### PR TITLE
2531 cleaning

### PR DESCRIPTION
**Pull request by ltello**
### Context
Remove legacy code, tables and fields. Card: [Post winter go-live tasks](https://trello.com/c/gdWooy3j)

### CODE REVIEW

**Review by nigel-lowry — APPROVED** [review]

**Comment by ltello** [comment]
[Post winter go-live tasks](https://trello.com/c/gdWooy3j/2531-post-winter-go-live-tasks)

File: app/models/responsible_body.rb [hunk 1]
```diff
@@ -1,25 +1,12 @@
 require 'computacenter/responsible_body_urns'
 
 class ResponsibleBody < ApplicationRecord
-  alias_attribute :sold_to, :computacenter_reference
```
Comment (markhallen) [inline app/models/responsible_body.rb:4]: Can we keep this please?

Comment (markhallen) [reply app/models/responsible_body.rb:4]: Never mind. I see it below 👍

**Review by markhallen — APPROVED** [review]
:-)

**Review by kenfodder — APPROVED** [review]

**Comment by ltello** [comment]
Thanks all for the approval

**Pull request merged**